layouts/partials/content.html

```diff
@@ -228,7 +228,7 @@ <h2>Enterprise Support</h2>
         <div class="join-us-wrap">
             {{ $cncf := resources.Get "/img/logo_cloudnative.png" }}
             <h2>We are a Cloud Native Computing Foundation sandbox project.</h2>
-            <img src="https://www.cncf.io/wp-content/uploads/2022/07/cncf-color-bg.svg" alt="opened box" loading="lazy"/>
+            <img src="https://www.cncf.io/wp-content/uploads/2022/07/cncf-white-logo.svg" alt="opened box" loading="lazy" width="200" height="auto"/>
```

Comment:
This looks a bit too small on large screens. Let's go with `300em` I think this fits better 

```suggestion
            <img src="https://www.cncf.io/wp-content/uploads/2022/07/cncf-white-logo.svg" alt="opened box" loading="lazy" width="300em" height="auto"/>
```